cli/cli-client/src/main/java/org/infinispan/cli/impl/CliRuntimeRunner.java

```diff
@@ -0,0 +1,114 @@
+package org.infinispan.cli.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.aesh.command.AeshCommandRuntimeBuilder;
+import org.aesh.command.Command;
+import org.aesh.command.CommandException;
+import org.aesh.command.CommandNotFoundException;
+import org.aesh.command.CommandRuntime;
+import org.aesh.command.impl.registry.AeshCommandRegistryBuilder;
+import org.aesh.command.invocation.CommandInvocation;
+import org.aesh.command.parser.CommandLineParserException;
+import org.aesh.command.registry.CommandRegistryException;
+import org.aesh.command.validator.CommandValidatorException;
+import org.aesh.command.validator.OptionValidatorException;
+
+public class CliRuntimeRunner {
```

Comment:
is this class be used for something more than batching? 
It seems to have more stuff in it then it needs.